client/src/index.js

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-
+import {BrowserRouter as Router, Route} from 'react-router-dom';
```

Comment:
You don't really need to import `Route` here, but that's just me being nitpicky.